src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -107,6 +122,7 @@ class YarnBuilder extends AbstractBuilder {
     }
     finally {
       steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'functional-output/**/*'
+      savePodLogs("mutation")
```

Comment:
mutation doesn't run against remote state so this doesn't make sense as far as I know
```suggestion
```